lib/livebook/intellisense.ex

```diff
@@ -439,6 +444,16 @@ defmodule Livebook.Intellisense do
     end
   end
 
+  defp format_meta(:deprecated, %{deprecated: deprecated}) do
+    "This function is deprecated. " <> deprecated
```

Comment:
I'd make the indication stronger, something like:

```suggestion
    "**Deprecated**. " <> deprecated
```

Reply:
Done.